e2e_tests/test_darwin.py

```diff
@@ -63,6 +95,55 @@ def test_darwin_push(local_dataset_with_images: E2EDataset) -> None:
     )
     assert result[0] == 0
 
+def test_darwin_import(local_dataset_with_annotations: E2EDataset) -> None:
+    """
+    Test importing a dataset via the darwin cli, dataset created via fixture with annotations added to objects
+    """
+    assert local_dataset_with_annotations.id is not None
+    assert local_dataset_with_annotations.name is not None
+    assert local_dataset_with_annotations.directory is not None
+    result = run_cli_command(
+        f"darwin dataset push {local_dataset_with_annotations.name} {local_dataset_with_annotations.directory}"
+    )
+    assert result[0] == 0
+    sleep(2)
+    result = run_cli_command(
+        f"yes Y | darwin dataset import {local_dataset_with_annotations.name} darwin {Path(local_dataset_with_annotations.directory) / 'annotations'}"
+    )
+    assert result[0] == 0
+
+
+def test_darwin_export(local_dataset_with_annotations: E2EDataset, config_values: ConfigValues) -> None:
+    """
+    Test exporting a dataset via the darwin cli, dataset created via fixture with annotations added to objects
+    """
+    assert local_dataset_with_annotations.id is not None
+    assert local_dataset_with_annotations.name is not None
+    assert local_dataset_with_annotations.directory is not None
+    result = run_cli_command(
+        f"darwin dataset push {local_dataset_with_annotations.name} {local_dataset_with_annotations.directory}"
+    )
+    assert result[0] == 0
+    sleep(2)
+    result = run_cli_command(
+        f"yes Y | darwin dataset import {local_dataset_with_annotations.name} darwin {Path(local_dataset_with_annotations.directory) / 'annotations'}"
+    )
+    assert result[0] == 0
+    
+    # Get class ids as export either needs a workflow and complete annotations or the class ids
+    url = f"{config_values.server}/api/teams/{config_values.team_slug}/annotation_classes?include_tags=true"
+    response = api_call('get', url, None, config_values.api_key)
+    if not response.ok:
+        raise Exception(f"Failed to get annotation classes: {response.text}")
+    classes = response.json()['annotation_classes']
+    class_ids = [c["id"] for c in classes]
+    class_str = " ".join([str(c) for c in class_ids])
+    # Test darwin export
+    sleep(2)
+    result = run_cli_command(
```

Comment:
I think it would be good to check if the export was actually created (not necessarily completed). I think you can do that with `releases` CLI command.

Reply:
A little easier said than done, darwin-py truncates command line output if it's too long. Oddly, testing in debug environment exhibits different behaviour than running with pytest command. I think the cli output library we have might be influenced by local terminal settings on truncation size.

Reply:
as in, an assert "test_darwin_export" in output can both pass and fail depending on where it's run, export name is truncated below for instance when run via pytest command, but passes when I run this in the debugger
![image](https://github.com/v7labs/darwin-py/assets/9366646/75cb2b78-2048-488b-b741-3ca9474fcd2f)

Reply:
Checking that there's not no releases will have to do

Reply:
I guess for now... 
But eventually we'll need to figure this one out. Maybe special test mode where output is unformatted?

Reply:
How does this output work when you redirect it to file?

Reply:
Unfortunately the same result, the output is truncated in the file too, but also differs based on if you run from local command line or in debugging mode.

Reply:
Rich has a formatter class that you can pass to it, that overrides the formatting - there's probably a plainer one that could be used for specific cases if needed, but we'd have to make changes to cli code to allow us to switch to it, which is changing the code to suit the test a bit.

Reply:
Theoretically, you can create a virtual VT of any display width, and pipe to it, but that's a lot of work for a small gain I think.

Reply:
I think we need a `--porcelain` type command that allows darwin to run simple formatted output.